cmd/agent/app/snmp.go

```diff
@@ -314,3 +320,19 @@ func printValue(pdu gosnmp.SnmpPDU) error {
 
 	return nil
 }
+
+var strippableSpecialChars = map[byte]bool{'\r': true, '\n': true, '\t': true}
+
+// IsStringPrintable returns true if the provided byte array is only composed of printable characeters
+func IsStringPrintable(bytesValue []byte) bool {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think we should import it from from https://github.com/DataDog/datadog-agent/blob/daa9cb2b8aa5148db1e7cb733d3bcaf56b852980/pkg/snmp/gosnmplib/gosnmp_utils.go#L28-L40 
to avoid duplication and keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).